docs/guides/redis/reconfigure-tls/sentinel.md

```diff
@@ -389,20 +362,6 @@ rd-ops-rotate    ReconfigureTLS   Successful    5m5s
 
 We can see from the above output that the `RedisOpsRequest` has succeeded. 
 
-Now, let's check the expiration date of the certificate.
-
-```bash
-$ kubectl exec -it -n demo rd-sample-0 -c redis -- bash
-
-root@rd-sample-0:/data# apt-get update; apt-get install openssl;
-...
-
-root@rd-sample-0:/data# openssl x509 -in /certs/server.crt -inform PEM -enddate -nameopt RFC2253 -noout
-notAfter=May 10 06:04:12 2023 GMT
-```
-
-As we can see from the above output, the certificate has been rotated successfully.
-
 ### Create RedisSentinelOpsRequest
 
 Now we are going to increase it using a RedisOpsRequest. Below is the yaml of the ops request that we are going to create,
```

Comment:
Now we are going to increase it using a RedisOpsRequest -> RedisSentinelOpsRequest
what are we referring by it? since we have removed the expiration time checking part.